contracts/CertiKSecurityOracle.sol

```diff
@@ -27,11 +27,23 @@ contract CertiKSecurityOracle is Ownable {
   constructor() public {
     initialize();
   }
+  
+  function isContract(address _addr) private view returns (bool) {
+    uint32 size;
+    assembly {
+      size := extcodesize(_addr)
+    }
+    return (size > 0);
+  }
 
   function getSecurityScore(
       address contractAddress,
       bytes4 functionSignature
     ) public view returns (uint8) {
+    if (!isContract(contractAddress)) {
```

Comment:
Perhaps not embed in the score function because external addresses can be in blacklist.

Reply:
We are not able to tell if an address if blacklisted or not because we can only get a score.

Do you think an address can cause damage to a contract if it's not a contract?

Reply:
May be other usecase such as not allowing addresses in blacklist to continue interact with the contract. Check the `msg.sender` for example. Not sure if something to be considered at this stage.

Reply:
Yeah maybe we can create a blacklist in the contract, or we should update the score in the contract with 0 if it's in the blacklist. Otherwise we are not able to tell the difference between a low score and a blacklisted one.